legend-depot-pure-model-context/src/main/java/org/finos/legend/depot/services/pure/model/context/PureModelContextServiceImpl.java

```diff
@@ -81,11 +90,16 @@ protected String resolveAndValidateClientVersion(String clientVersion)
     }
 
     protected PureModelContextData getPureModelContextData(Stream<Entity> entities, String groupId, String artifactId, String versionId, String clientVersion)
+    {
+        return getPureModelContextData(entities, buildAlloySDLC(groupId, artifactId, versionId), clientVersion);
```

Comment:
rename this to buildPureModelContextData

Reply:
changed to build